src/main/java/PageObjects/BasesLocationDetailsPage.java

```diff
@@ -21,10 +21,14 @@ public BasesLocationDetailsPage(WebDriver driver) {
         webDriverWait = new WebDriverWait(driver, Duration.ofSeconds(7));
     }
 
-    public String copyBaseAddress(){
-        WebElement addressRow = driver.findElement(locationOfBase);
-        String value = addressRow.getText();
-        return value;
+    public boolean isKoluszkiDetailDisplayed() {
+        boolean logicValue = false;
+        WebElement webElement = driver.findElement(By.cssSelector(".tytul-obiekt"));
+        String valueOfBase = webElement.getText();
+        if (valueOfBase.equals("Baza Paliw nr 1 w Koluszkach")) {
+            logicValue = true;
+        }
+        return logicValue;
```

Comment:
You can simplify this method to one line:
return koluszkiBaseBanner.getText().equals("Baza Paliw nr 1 w Koluszkach");